lib/charms/layer/azure.py

```diff
@@ -179,6 +180,17 @@ def enable_network_management(request):
     _assign_role(request, StandardRole.NETWORK_MANAGER)
 
 
+def enable_loadbalancer_management(request):
+    """
+    Enable network management for the given application.
+    """
+    log('Enabling load balancer management')
+    _assign_role(request, _get_role('lb-manager'))
+    # In this case, we need to have permissions on both VM and network RGs
+    if hookenv.config('vnetResourceGroup') != request.resource_group:
+        _assign_role(request, _get_role('lb-manager'), hookenv.config('vnetName'))
```

Comment:
I'm not quite following this - is `vnetName` also usable as a Resource Group? (I'm not very familiar with Azure)

Reply:
erh... Typo, gonna fix it